### DISCUSSION TITLE
Using Vsensor as input to second Vsensor

### DISCUSSION THREAD

**Discussion by hvontres** [tjko/fanpico#77] [Q&A]
I was wondering if it is possible to configure a Vsensor to use another Vsensor as one of the inputs? If so, what would the syntax for the configure command be? Reading the command reference, it seems that only actual sensors can be used. I would like to set up a vsensor to use the difference between a temperature reading and a setpoint to control fan speed. I guess I could use the Toffset for this as well, but it would be nice to still have the original temperature reading avialable.

**Comment by tjko** [comment]
Do you mean that [CONFigure:VSENSORx:SOURCE](https://github.com/tjko/fanpico/blob/main/commands.md#configurevsensorx-commands) command would support both sensors and vsensors as source for the calculations (min, max, avg, delta)?

I had not thought about that use case before, it should be relatively easy to add support for it.   One option would be to support mixing and matching SENSORS and VSENSORS by adding 100 to the VSENSOR number, so sensors would be 1,2,3, and vsensors would be 101,102,..,108.

**Reply by tjko** [reply]
I added experimental support: https://github.com/tjko/fanpico/commit/8a7ec21b9c0ece074c017d36d70882039bbac064

If you want to test, you can find firmware images here: https://github.com/tjko/fanpico/actions/runs/7955759913?pr=78
